tests/circuit_graph/test_circuit_graph.py

```diff
@@ -211,9 +211,9 @@ def test_diagonalizing_gates(self):
     def test_is_sampled(self):
         """Test that circuit graphs with sampled observables properly return
         True for CircuitGraph.is_sampled"""
-        circuit = CircuitGraph([qml.expval(qml.PauliX(0)), qml.var(qml.PauliZ(1))], {})
+        circuit = CircuitGraph([qml.expval(qml.PauliX(0)), qml.var(qml.PauliZ(1))], {}, Wires([0, 1]))
         assert not circuit.is_sampled
 
-        circuit = CircuitGraph([qml.expval(qml.PauliX(0)), qml.sample(qml.PauliZ(1))], {})
+        circuit = CircuitGraph([qml.expval(qml.PauliX(0)), qml.sample(qml.PauliZ(1))], {}, Wires([0, 1]))
```

Comment:
This addition must have been a pain to change everywhere!

Reply:
There were worse pains along this PR ;)